README.md

```diff
@@ -27,32 +27,39 @@ In AWS Web Console, add a 'CloudWatch Logs' trigger for the Lambda function. In
 #### 2. CloudTrail Logs
```

Comment:
1. Add new way of uploading zip for creating lambda
2. Add readme instructions for new AWS services introduced